index.html

```diff
@@ -31,141 +42,167 @@ <h1 class="title">I’m Jesse<br> Glad to see<br> you!</h1><br><br>
-            <img class="lang-img" src="./images/css.png" alt="css"> 
+            <img class="lang-img" src="./images/css.png" alt="css">
             <span class="span">CSS</span>
           </li>
         </ul>
         <div class="frame">
-         <h1 class="lg">Frameworks</h1>
-         <img class="forward" src="./images/20.png" ></img>
-         <hr class="divider1">
-         <h1 class="lg">Skills</h1>
-         <img class="forward" src="./images/20.png" ></img>
-         <hr class="divider2">
+          <h1 class="lg">Frameworks</h1>
+          <img class="forward" src="./images/20.png"></img>
+          <hr class="divider1">
+          <h1 class="lg">Skills</h1>
+          <img class="forward" src="./images/20.png"></img>
+          <hr class="divider2">
         </div>
-      </section>
-      <footer class="contact">
-        <h2 class="contact-me"><b>Contact me</b></h2>
-        <p class="more">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-        <form action="https://formspree.io/f/mrgvqyqz" method="post">
-          <label for="full-name">
-            <input type="text" class="input-text" name="full_name" id="fullname" placeholder="Enter your full name" maxlength="30" required>
-          </label>
-          <label for="email_address">
-            <input type="email" class="input-email" name="email_address" id="email_address" placeholder="Enter your email address" required>
-          </label>
-          <label for="text_message">
-            <textarea name="message" id="text-message" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>
-          </label>
-          <button id="contact-btn" class="btn contact-btn" type="submit"> Get in touch</button>
+
+      </div>
     </section>
+    <footer class="contact">
+      <h2 class="contact-me"><b>Contact me</b></h2>
+      <p class="more">If you have an application you are interested in developing, a feature that you need built or a
+        project that needs coding. I'd love to help with it </p>
+      <form action="https://formspree.io/f/mrgvqyqz" method="post">
+        <label for="full-name">
+          <input type="text" class="input-text" name="full_name" id="fullname" placeholder="Enter your full name"
+            maxlength="30" required>
+        </label>
+        <label for="email_address">
+          <input type="email" class="input-email" name="email_address" id="email_address"
+            placeholder="Enter your email address" required>
+        </label>
+        <label for="text_message">
+          <textarea name="message" id="text-message" cols="30" rows="10" placeholder="Write your message here"
+            maxlength="500" required></textarea>
+        </label>
+        <button id="contact-btn" class="btn contact-btn" type="submit"> Get in touch</button>
+        </form>
+      </footer>
```

Comment:
- The mobile version of the contact form does not look like the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your project:
<img width="281" alt="ndgnzdn" src="https://user-images.githubusercontent.com/73607512/221367363-e3e40f48-2938-407a-b883-1887115214d2.PNG">